hook.js

```diff
@@ -191,6 +191,22 @@ async function processModule ({
       continue
     }
 
+    if (NODE_MAJOR >= 15) {
+      // const modName = `v${Buffer.from(n, 'utf-8').toString('hex')}`
+      const modName = normalizeModName(n)
```

Comment:
Not sure what the appropriate / preferred method of normalization is here. I'll update this given some guidance.